README.md

```diff
@@ -20,6 +21,78 @@ The following file types are known to be _writable_ with this library:
 - AnaMark multiple scales tuning (_.msf_)
 - HTML (for scale distribution and sharing)
 
+## Building the Library and Tests as Standalone Assets
+
+The AnaMark Tuning Library uses CMake to configure and build the solution.
+CMake is a widely available open source build configuraiton tool which can 
+create build assets for the development environment of your choice. You can
+install it from [cmake.org](https://cmake.org); if you install VS2017 or 2019
+you already have it; you can easily install it with homebrew on mac or apt on
+linux. 
+
+But once it is installed the recipe is simple. Run cmake to make your build
+assets, and either then open those build assets or run cmake to run a build.
+
+### Windows
+
+Open a Visual Studio command prompt. Check out this code and change to the
+directory. Type
+
+```
+cmake -Bbuild
+```
+
+and you will have in your build directory a visual studio solutions file which
+will allow you to create the library or test targets or you can build on the 
+command line with
+
+```
+cmake --build build --config Release
+```
+
+### macOS
+
+Run CMake to create xcode assets. For example
+
+```
+cmake -Bbuild -GXcode
+```
+
+then you can build in xcode or from the command line with
+
+```
+cmake --build build --config Release 
+```
+
+Command line builds are greatly improved by piping into 'xcpretty' utility.
```

Comment:
What is _xcpretty_, and how does it "improve" a command line build?  Provide an example of "piping" the build into xcpretty and explain why it improves it.

Reply:
Just force-pushed a branch with a clarifying change. (short version: xcpretty makes the command line Xcode output readable and is installable from a link I should have included in the doc). Thanks!